user/views.py

```diff
@@ -18,7 +18,6 @@ class CreateTokenView(ObtainAuthToken):
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
```

Comment:
i think we don't need CreateTokenView in this task, couse we don't use it

Reply:
Agree with @mdubyna , you can remove this view + imports related to this view from this file + remove from users/urls.py this createtokenview import